### DISCUSSION TITLE
Learning Git

### DISCUSSION THREAD

**Discussion by unknownguy1805** [community/community#132989] [New to GitHub] [answered]
What's the best way to learn Git for a beginner?

**Comment by unknownguy49** [comment, marked as answer]
The best way to learn Git is by using it! Start with basic commands like git init, git add, git commit, and git push. Follow tutorials, watch videos online and create demo repositories with basic files to see the github flow. Practicing with small projects also helps a lot.

**Comment by intondiscott** [comment]
I would say just Google the most commonly used commands such as add, clone, status, push/pull, etc. Learn about feature branching too as this will help with version control in case a revision goes wrong.

**Comment by JuneLin2001** [comment]
You can use Github desktop first! And use it to know some feature (like push、pull、clone...etc) I think GUI is friendly for beginners.
